src/python/pants/backend/docker/goals/package_image.py

```diff
@@ -187,6 +210,82 @@ def get_context_root(self, default_context_root: str) -> str:
         return os.path.normpath(context_root)
 
 
+@dataclass(frozen=True)
+class ImageRefRegistry:
+    registry: DockerRegistryOptions | None
+    repository: str
+    tags: tuple[ImageRefTag, ...]
+
+
+@dataclass(frozen=True)
+class ImageRefTag:
+    template: str
+    formatted: str
+    full_name: str
+
+
+@dataclass(frozen=True)
+class DockerInfoV1:
+    """The format of the .docker-info.json file."""
```

Comment:
Why a leading period on the file name?

Reply:
nvm, noticed that this is the file ending used for the output file.

Reply:
Yeah, that's right. I'll tweak it to be clearer.